Chaincase.UI/ViewModels/StatusViewModel.cs

```diff
@@ -46,14 +47,15 @@ public class StatusViewModel : ReactiveObject
         private bool _downloadingBlock;
         private StatusSet ActiveStatuses { get; }
 
-        public StatusViewModel(Global global, ChaincaseWalletManager walletManager, Config config, UiConfig uiConfig, BitcoinStore bitcoinStore, ChaincaseSynchronizer synchronizer)
+        public StatusViewModel(Global global, ChaincaseWalletManager walletManager, Config config, UiConfig uiConfig, BitcoinStore bitcoinStore, ChaincaseSynchronizer synchronizer, P2EPServer P2EPServer)
```

Comment:
As stated earlier in the review, flashing screen or dep from the main StatusViewModel? Why?